src/components/order-service/order-service.controller.ts

```diff
@@ -174,7 +196,17 @@ export class OrderServiceController implements angular.IController {
 
     this.validityWatcher = this.$scope.$watch("$ctrl.forms.bindForm.$valid", (isValid: any, lastValue: any) => {
       this.bindStep.valid = isValid;
-      this.reviewStep.allowed = this.bindStep.valid;
+      this.bindParametersStep.allowed = !this.bindParametersStep.hidden && isValid;
```

Comment:
nit: don't really need to check hidden here, if it is hidden then allowed will never be used.